### ISSUE TITLE
Eval duration

### ISSUE THREAD

**Issue by david-rohrschneider** [UX-Decoder/Segment-Everything-Everywhere-All-At-Once#86]
Hello, i am running the training script with the default config of seem focal t.

I use a custom dataset in coco format and it actually works, but i was wondering about why one training epoch needs 10min on my single GPU Environment and the corresponding evaluation part needs 15-20min. 
Is this the default behaviour because the evaluation script is generally slower to perform? Or is this unusual?

The split is 2250 train / 750 test and batch size is 8 for both.

**Comment by MaureenZOU** [comment]
The problem with that is probably caused by we only support evaluation of one image at one GPU, because of padding problem. You could change the code for that part and re-evaluate, although the performance would be slightly worse, but it would be faster.
